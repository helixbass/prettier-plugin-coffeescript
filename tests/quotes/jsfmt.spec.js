run_spec(__dirname, ['coffeescript'], {singleQuote: true})
run_spec(__dirname, ['coffeescript'], {singleQuote: false})
