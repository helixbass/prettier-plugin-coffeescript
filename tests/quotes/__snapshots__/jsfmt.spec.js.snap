// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`heredoc.coffee 1`] = `
a = """
  b
"""

a = '''
  b
'''

a = """
  #{b}
"""

a = '''
  #{b}
'''

a = "
  #{b}
"

a = '
  #{b}
'

a = """
  '''
"""

a = '''
  """
'''

a = """
  '"""

a = '''
  "'''

a = """
  '
"""

a = '''
  "
'''

a = """'
  """

a = '''"
  '''
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
a = '''
  b
'''

a = '''
  b
'''

a = """
  #{b}
"""

a = '''
  #{b}
'''

a = "
  #{b}
"

a = '
  #{b}
'

a = """
  '''
"""

a = '''
  """
'''

a = """
  '"""

a = '''
  "'''

a = '''
  '
'''

a = '''
  "
'''

a = ''''
  '''

a = '''"
  '''

`;

exports[`heredoc.coffee 2`] = `
a = """
  b
"""

a = '''
  b
'''

a = """
  #{b}
"""

a = '''
  #{b}
'''

a = "
  #{b}
"

a = '
  #{b}
'

a = """
  '''
"""

a = '''
  """
'''

a = """
  '"""

a = '''
  "'''

a = """
  '
"""

a = '''
  "
'''

a = """'
  """

a = '''"
  '''
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
a = """
  b
"""

a = """
  b
"""

a = """
  #{b}
"""

a = '''
  #{b}
'''

a = "
  #{b}
"

a = '
  #{b}
'

a = """
  '''
"""

a = '''
  """
'''

a = """
  '"""

a = '''
  "'''

a = """
  '
"""

a = """
  "
"""

a = """'
  """

a = """"
  """

`;
